Unix/samples/Providers/Fan/common/common.c

```diff
@@ -92,7 +92,7 @@ Fan fans[] =
         "ABC_Fan",
         "linux-22",
         "FAN:3", /* InstallDate */
-        "CHASIS FAN", /* ElementName */
+        "CHASSIS FAN", /* ElementName */
```

Comment:
I presume this isn't a problem
https://forums.tomshardware.com/threads/chassis-fan-vs-cpu-fan.1464470/